### PR TITLE
Fix #440 Feedly hints

### LINKED ISSUE #440: No hints for feedly.com
Hi.
Hints are not generated for most of feedly.com
I use VimFX dev. channel build I compiled on 2015-01-19, Win 8.1, Firefox beta 36

### PR DESCRIPTION
**Pull request by TheGreenHabi**
Fix #440. Fixed generation of hints for feedly.com by adding function to check for this element attributes:
```
data-app-action
data-uri
data-page-action
```

### CODE REVIEW

File: extension/lib/commands.coffee [hunk 1]
```diff
@@ -372,6 +374,7 @@ command_marker_yank = (vim) ->
   filter = (element, getElementShape) ->
     type = switch
       when isProperLink(element)       then 'link'
+      when isDataApp(element)          then 'clickable'
```
Comment (lydell) [inline extension/lib/commands.coffee]: Remove this line.

File: extension/lib/commands.coffee [hunk 2]
```diff
@@ -288,7 +288,9 @@ command_follow = (vim, event, count) ->
            element.hasAttribute('oncommand') or
            element.getAttribute('role') in ['link', 'button'] or
            # Twitter special-case.
-           element.classList.contains('js-new-tweets-bar')
+           element.classList.contains('js-new-tweets-bar') or
+           # Feedly special-case.
+           isDataApp(element)
```
Comment (lydell) [inline extension/lib/commands.coffee]: Inline `isDataApp`.

**Comment by lydell** [comment]
Looks good! Please fix the things I commented on and the squash everything into one commit with a nice commit message.

File: extension/lib/commands.coffee (hunk 2 above)
Comment (TheGreenHabi) [inline extension/lib/commands.coffee]: "To inline" means to make it this way?
````
isDataApp(element) # Feedly special-case.
```

**Comment by TheGreenHabi** [comment]
@lydell, what is your opinion about changing `isDataApp` function name to `hasAttribute`?
This way it will be self explanatory and not so vague. And it will be ready for other attributes from different website.

**Pull request closed without merging**